palindrome/palindrome.ts

```diff
@@ -0,0 +1,9 @@
+const palindrome = (str: string) => {
+  const reverseStr = str.split('')
+    .reverse()
```

Comment:
Pretty strange transposition. Please, keep it in one line.

Reply:
done